src/AsyncSubject.ts

```diff
@@ -26,8 +24,10 @@ export class AsyncSubject<T> extends Subject<T> {
   }
 
   next(value: T): void {
-    this.value = value;
-    this.hasNext = true;
+    if (!this.hasCompleted) {
```

Comment:
replicating same behavior of RxJS4, do not allow change values once it's completed. (https://github.com/Reactive-Extensions/RxJS/blob/29f85627e4ab71b050480f26dcc5d737f1b840f4/src/core/subjects/asyncsubject.js#L95)